src/generators/web/build/css.mjs

```diff
@@ -0,0 +1,62 @@
+import { readFile } from 'node:fs/promises';
+
+import { bundleAsync } from 'lightningcss';
+
+/**
+ * Rolldown plugin to compile `.module.css` files using lightningcss
+ * and emit a single bundled CSS file.
+ *
+ * TODO(avivkeller): Once Rolldown supports CSS Modules natively, we
+ * can remove this
+ *
+ * @returns {import('rolldown').Plugin}
+ */
+export default () => {
+  const cssChunks = new Set();
+
+  return {
+    name: 'css-loader',
+
+    /**
+     * Handles loading and transforming CSS module files
+     * @param {string} id - The file path being loaded
+     */
+    async load(id) {
```

Comment:
Recommendation: To speed up the build I'd use a [hook filter](https://rolldown.rs/guide/plugin-development#plugin-hook-filters) here that excludes all files that aren't `.module.css` (or includes all of them). This should reduce rust<>js communication overhead